### ISSUE TITLE
Version 3.8.6.1 - Move > Extras > Normalize and Consolidate broken

### ISSUE THREAD

**Issue by edmiller47** [dasilva333/TowerGhostForDestiny#565]
The Normalize and Consolidate functions aren't working properly. 
For example: I have 5 motes of light on each of three characters and several hundred in the vault. If I select the motes on character 2 and consolidate, the motes disappear from the other two characters as expected, but the motes in the vault disappear too! Refreshing the vault then will display the correct number of motes.
Then if I select the 15 motes on character 2 and normalize across the three characters, I see 5 motes on character 1, 0 on character 2 and 3, and 0 in the vault. On refresh, there are 5 motes on character 1 and 10 on character 2, 0 on character 3, and the rest in the vault.

**Comment by dasilva333** [comment]
Thanks man I'll get it fixed

**Comment by edmiller47** [comment]
Don't know if this is a side issue or a bug of its own - but for the third time in the last 4 or 5 days, the Chrome version of Tower Ghost on my laptop has reverted itself back to 3.8.2.1. I had to toggle Auto Updates off and on to get back to 3.8.6.1. I had opened TG a few hours ago, edited a couple of loadouts, and then left the computer idle. When I came back to it I noticed the earlier version was there.

I need to reboot my computer to complete a software update. I'll report back if this continues to occur after restart.
